src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1263,6 +1296,21 @@ public void TestCapitalizeV21()
                 syntax: context.SyntaxCompatibilityLevel);
         }
 
+        [Test]
+        public void TestCapitalizeV21a()

```

Comment:
nit: Shopify spec also has as example {{ "title" | capitalize }} and {{ 'capitalize me' | capitalize }}
https://shopify.github.io/liquid/filters/capitalize/
https://shopify.dev/docs/themes/liquid/reference/filters/string-filters#capitalize